integ-tests/src/cucumber/java/steps/dataservice/DescribeEnvironmentSteps.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package steps.dataservice;
+
+import com.amazonaws.blox.dataservicemodel.v1.model.Environment;
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentId;
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentType;
+import com.amazonaws.blox.dataservicemodel.v1.model.InstanceGroup;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.CreateEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DescribeEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.UpdateEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DeleteEnvironmentRequest;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.CreateEnvironmentResponse;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.DescribeEnvironmentResponse;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.UpdateEnvironmentResponse;
+
+import configuration.CucumberConfiguration;
+import cucumber.api.java8.En;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import steps.wrappers.DataServiceWrapper;
+import static org.junit.Assert.assertTrue;
+
+import java.util.UUID;
+
+@ContextConfiguration(classes = CucumberConfiguration.class)
+public class DescribeEnvironmentSteps implements En {
+
+  @Autowired private DataServiceWrapper dataServiceWrapper;
+  private static final String ACCOUNT_ID = "accountID";
```

Comment:
this will fail when we add input validation since accountids need to be 12 digit numbers. We can use the accountId that the tests are going to run as. for now, we can just use 12345678912

Reply:
Sure! good to know! I'll change this.

Reply:
Just realize that 12345678912 is only 11 digit. I'll use 123456789012 instead.